README.md

```diff
@@ -2,6 +2,7 @@
   <img src="https://assets.vercel.com/image/upload/v1588805858/repositories/vercel/logo.png" height="96">
   <h3 align="center">Vercel</h3>
```

Comment:
Should we make this header the link instead?

Reply:
Agreed, we could make the name and image link to the website.